filament/backend/src/metal/MetalBuffer.mm

```diff
@@ -22,7 +22,7 @@
 namespace filament {
 namespace backend {
 
-std::atomic<uint64_t> TrackedMetalBuffer::aliveBuffers = 0;
+std::array<uint64_t, TrackedMetalBuffer::TypeCount> TrackedMetalBuffer::aliveBuffers = { 0 };
```

Comment:
so you sure this is never accessed from multiple threads?

Reply:
Yes, this is always accessed from the driver thread